python/examples/profiling.py

```diff
@@ -1,14 +1,16 @@
+"""Profiling of the Python API."""
```

Comment:
That's an example of how to use the internal profiler to get some timing data on the calculation of SOAP features. The Python API is mostly unrelated to this profiler